docker-compose.ci.yml

```diff
@@ -0,0 +1,9 @@
+tests:
+  build: .
```

Comment:
Because the first step of the pipeline builds the image and pushes it to quay.io you can avoid rebuilding it in tests I believe by specifying that quay.io image here
`image: "${DOCKER_REPOSITORY}:${COMMIT}"`

Thoughts?